### ISSUE TITLE
Website localization, or even folders 'diagnostics'

### ISSUE THREAD

**Issue by SergeyZhurikhin** [code-cracker/code-cracker.github.io#17]
What are the proposals for the organization of links to localized pages.
For example:
code-cracker.github.io/**en**/diagnostics - for English
code-cracker.github.io/**ru**/diagnostics - for Russian language.
Then it is necessary to alter the method:

``` CS
public static class HelpLink
{
    public static string ForDiagnostic(DiagnosticId diagnosticId) =>
        $"https://code-cracker.github.io/diagnostics/{diagnosticId.ToDiagnosticId()}.html";
}
```

Like that:

``` CS
public static string ForDiagnostic(DiagnosticId diagnosticId)
{
    return $"https://code-cracker.github.io/diagnostics/{CultureInfo.CurrentCulture.Parent.Name}/{diagnosticId.ToDiagnosticId()}.html";
}
```

**Comment by carloscds** [comment]
@SergeyZhurikhin It's a good point. Do you have interest in translate documentation for Russian ?

**Comment by giggio** [comment]
I like it!

**Comment by SergeyZhurikhin** [comment]
Yes, in addition to high-quality translation of resource strings in any case have to go to each page of the document and make sure that I understand the line resource.
If there are no objections, I will do so

code-cracker.github.io/**en**/diagnostics - for English
code-cracker.github.io/**ru**/diagnostics - for Russian language.
code-cracker.github.io/diagnostics - apparently, for some time, this folder will remain, since the new version does not directly flow into the user's computer. After some piece time to remove it.

In tandem with this change, I change HelpLink.

**Comment by giggio** [comment]
@SergeyZhurikhin Before we do that, it would be nice to have a fallback to English. If a diagnostic page does not exist for Russian, for example, it would be nice to be able to show the English one. How would we do that?

**Comment by SergeyZhurikhin** [comment]
@giggio In the first stage, I would like to copy all pages from **diagnostics**  to **ru/diagnostics**, and leisurely to translate into Russian. Thus the page in any case, will be available.

And if you ask when there are no pages like this: http://code-cracker.github.io/diagnostics/CC0095.html
http://code-cracker.github.io/diagnostics/CC0105.html 
There will have to create the page, and may create a console application - synchronizer, that checks the pages and the rules in the code.

**Comment by giggio** [comment]
Ok, it makes sense that you start translating to ru. You can go ahead and start translating them, and sending the PRs.

To solve the problem, we could keep sending the user to `/`, and sniff the browser language (the `Accept-Language` header) and automatically redirect them to `/ru` if it is the case. How about that?

More on that: http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.4

I don't know how we would do that with Jekyll and Github pages, or maybe using JavaScript. It would be a good idea to research that.

**Comment by SergeyZhurikhin** [comment]
@giggio We can experiment with redirect.

**Comment by giggio** [comment]
Yeah, but I don't know if we can do that. We will have to look it up.
